cadence/transactions/flow/mint_tokens.cdc

```diff
@@ -18,7 +18,7 @@ transaction(recipient: Address, amount: UFix64) {
     }
 
     execute {
-        let minter <- self.tokenAdmin.createNewMinter()
```

Comment:
Good catch! We should definitely merge this change. Could you remove the change in `FlowToken.cdc` and keep this one? Then we can merge this PR.

Reply:
I'll check it!

Reply:
i fixed it